### PR TITLE
Tyler init

### PR DESCRIPTION
**Pull request by lumsdent**
Initial riot api fetch

### CODE REVIEW

File: process_match_information.py [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+    # Fetch data from the API if match ID is provided
+    else:
+        match_data = fetch_match_data("NA1_" + match_id)
+    
     match_information = process_match_data(match_data)
-
+    
     # Output the processed data to a new file
-    output_file_path = file_name + "_process"
+    output_file_path = f"results/{(file_name, match_id)[match_id != ""]}_process.json"
```
Comment (Cerelius) [inline process_match_information.py:25]: I like adding the match_id here

File: process_match_information.py [hunk 2]
```diff
@@ -1,26 +1,45 @@
...
     with open(output_file_path, "w") as output_file:
         json.dump(match_information, output_file)
+
+
+def fetch_match_data(match_id):
+    api_key = os.getenv("RIOT_API_KEY")
+    headers = {"X-Riot-Token": api_key}
+    url = f"https://americas.api.riotgames.com/lol/match/v5/matches/{match_id}"
+    response = requests.get(url, headers=headers)
+    return response.json()
```
Comment (Cerelius) [inline process_match_information.py:35]: Great!

File: results/4945781121_process.json
Comment (Cerelius) [inline results/4945781121_process.json:1]: Do you want these committed?

File: results/match_information_response_process.json
Comment (Cerelius) [inline results/match_information_response_process.json:1]: Do you want these committed?

**Review by Cerelius — APPROVED** [review]

File: results/match_information_response_process.json
Comment (lumsdent) [reply results/match_information_response_process.json:1]: These specifically, not really, but I figured we could house the data generated in the repo.

**Pull request merged**